queries/javascript/locals.scm

```diff
@@ -6,6 +6,10 @@
 (arrow_function) @scope
 (function_declaration) @scope
 (method_definition) @scope
+(if_statement) @scope
+(for_statement) @scope
+(switch_statement) @scope
```

Comment:
I don't believe a switch statement actually creates a scope. You can create one in the case but that is covered by a `statement_block`

```js
case a: {
   // stuff
}
```